TAScheduler/UnitTests/tests.py

```diff
@@ -2946,3 +2946,94 @@ def test_none_to_remove(self):
     def test_remove(self):
         self.lecture.removeTA()
         self.assertIsNone(self.lecture.getLectureTAAsgmt(), "removeTA() did not remove from lecture")
+
+class UserEditMyContactInfoTest(TestCase):
+    def setUp(self):
+        # Create a user instance for testing
+        self.user = User.objects.create(
+            email_address='test@example.com',
+            password='safe_password',
+            first_name='Test',
+            last_name='User',
+            home_address='123 Main St',
+            phone_number=1234567890
+        )
+
+    def test_edit_contact_info(self):
+        # Simulate the edit operation
+        self.user.first_name = 'UpdatedName'
+        self.user.save()
+
+        # Fetch the updated user from the database
+        updated_user = User.objects.get(id=self.user.id)
+
+        # Check if the first name is updated
+        self.assertEqual(updated_user.first_name, 'UpdatedName')
+
+
+class UserGetContactInfoTest(TestCase):
+    def setUp(self):
+        # Create a user instance for testing
+        self.user = User.objects.create(
+            email_address='test@example.com',
+            password='safe_password',
+            first_name='Test',
+            last_name='User',
+            home_address='123 Main St',
+            phone_number=1234567890
+        )
+
+    def test_get_contact_info(self):
+        # Assume a method to retrieve user contact info
+        contact_info = {
```

Comment:
this method should call the .getContactInfo() method.

it might be good to move this dictionary  in the setup, call this method, and then check to make sure all of the fields are the same as this dictionary.